Client/src/Components/Game/Users/CardHeader/CardHeader.svelte

```diff
@@ -0,0 +1,18 @@
+<script>
```

Comment:
I think that the cardHeader should be in the common folder, or it shouldn't be named "CardHeader". Because in my opinion, CardHeader should be something reuseable, not only in the Users context. maybe we will use this card in other places?

Reply:
Hmm... I think I'll make "Card - Card header" type of thing a common, maybe w a custom slot

Reply:
yeah it sounds good